requirements-docs.txt

```diff
@@ -1,10 +0,0 @@
-biopython==1.65
```

Comment:
We'll still need `requirements-docs.txt`, since ReadTheDocs relies on it (for now...).